backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -0,0 +1,66 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.theme
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Typography
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.text.font.FontWeight
+import net.skyscanner.backpack.compose.tokens.BpkTypography
+
+val LocalBpkTypography = staticCompositionLocalOf {
+  BpkTypography()
+}
+
+@Composable
+fun BpkTheme(
+  typography: BpkTypography = BpkTheme.typography,
```

Comment:
I don't think typography should be customisable for the theme. Instead we could try to search for the fonts using string constants.

Reply:
We'll have to make it customizable to some extent - at least to override the font. i'm not sure what you mean with searching using string constants

Reply:
Can we take defaultFontFalimy here instead of entire typography?